pkg/info/server_info.go

```diff
@@ -0,0 +1,121 @@
+package info
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"runtime/debug"
+	"strings"
+	"time"
+)
+
+func GetSDKVersion() string {
+	version := "unknown"
```

Comment:
is an empty string a better value than "unknown"?

Reply:
updated.